chef_master/source/knife.rst

```diff
@@ -27,7 +27,7 @@ knife is a command-line tool that provides an interface between a local chef-rep
    * - :doc:`knife_using`
      - knife runs from a management workstation and sits in-between a Chef server and an organization's infrastructure.
    * - :doc:`knife_common_options`
-     - There are many options that are available for all knife subcommands.
+     - There are many command line options that are available for all knife subcommands.
```

Comment:
The introductory paragraph states that "knife is a command-line tool that provides..." I don't think it needs to be stated that the options are command line-based.